commands/token/parse.go

```diff
@@ -0,0 +1,147 @@
+package token
+
+import (
+	"context"
+
+	"github.com/fatih/structs"
+	"github.com/ionos-cloud/ionosctl/v6/internal/jwt"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/constants"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/core"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/printer"
+	authservice "github.com/ionos-cloud/ionosctl/v6/services/auth-v1"
+	"github.com/spf13/viper"
+)
+
+func TokenParseCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "token",
+		Resource:  "token",
+		Verb:      "parse",
+		Aliases:   []string{"p"},
+		ShortDesc: "Parse the contents of a Token",
+		LongDesc: `Use this command to parse a Token and find out Token ID, User ID, Contract Number, Role and Privileges (separate).
```

Comment:
this is a bit unclear: `Privileges (separate).`

How about

```
LongDesc: `Use this command to parse a Token and display its contents. By default, the command will output the Token ID, User ID, Contract Number, and Role associated with the token.

If you want to view the privileges associated with the token, you must set the --privileges flag. When this flag is set, the command will output a list of privileges instead of the default output.
```

Reply:
Will change it now.